### PR TITLE
Refactor media placeholder, making it a seperate component. DDFFORM-360

### PR DESCRIPTION
**Pull request by rasben**
Before now, we used various scss mixins to re-use the same placeholder component, that shows up when no image is available. Instead, we'll make a media-container component that can be re-used along with it's own CSS classes.

----

Please review it, by starting to look at the visual changes in chromatic :)
I've also tweaked some of the examples, so we see teasers without images.

-----

**This should not be merged, before we also have the CMS PR ready!**

https://github.com/danskernesdigitalebibliotek/dpl-cms/pull/861

### CODE REVIEW

**Comment by kasperbirch1** [comment]
It looks fine to me, but I think it would be good if the others from the team also looked at it.

It surprises me that the image is no longer 4/5 format here.
https://www.chromatic.com/test?appId=616ffdab9acbf5003ad5fd2b&id=65e868dd088ac5e6bee9c4c9

and that there is no crediting under the picture here
https://www.chromatic.com/test?appId=616ffdab9acbf5003ad5fd2b&id=65e868dd088ac5e6bee9c45d

Is it meant to be?

**Comment by rasben** [comment]
I know that this does not exactly follow the design, but, I've been unable to get the padding to "stick", when the caption is used.
I think compared to the wins of getting placeholder to work, and to re-using the media-container across all items, outweighs this "edgecase".

I've also tweaked some of the examples, to show heros without creditation, images etc.

**Comment by rasben** [comment]
>It surprises me that the image is no longer 4/5 format here.

It's because the text expands the whole box. I think that the huge spacing below on the old design is "wrong" - but, we can take the communication with DDF

**Review by LasseStaus — APPROVED** [review]
Approved 👍  

You didn't answer the comment for `.media-container--placeholder`. 

I think it would be decent to write a response for the 'refactor' comment

File: src/stories/Library/media-container/media-container.scss [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      @include typography($typo__card-placeholder);
+    }
+  }
+}
+
+@mixin media-container--small {
+  .media-container {
+    &--placeholder {
+      @include identity-placeholder($placeholder-paddings--small);
+    }
```
[the remark this answers is not in the document]
Comment (rasben) [reply src/stories/Library/media-container/media-container.scss:21]: Ah, sorry, i missed this comment:

I think our general "rule of thumb" is that we allow modifiers, but avoid it for seperate blocks

E.g.

```scss
.media-container {
  &--placeholder {} 🟢
  &__container {} 🔴
}
```

**Pull request merged**